components/auth/routes.js

```diff
@@ -49,6 +49,8 @@ function authApi(app) {
             typeOfUser,
           } = user;
 
+          const defaultPath = config.defaultPath[user.typeOfUser] || '/';
+
```

Comment:
In line 22 `next(boom.unauthorized('apiKeyToken is required'));` replace with `throw boom.unauthorized('apiKeyToken is required');`

Reply:
Done!